pages/1.10/deploying-services/uninstall/index.md

```diff
@@ -30,6 +29,8 @@ dcos package uninstall chronos
 
 ## Web interface
 
+<p class="message--note"><strong>NOTE: </strong>The Edge-LB service cannot be uninstalled using this procedure. Please follow the steps in the <a href="/services/edge-lb/1.2/uninstalling/">Edge-LB uninstall documentation</a>.</p>
```

Comment:
You are linking here to version 1.2, at some point we will have version 1.3, 1.4, etc... Shouldn't we mention here something like: `Please follow the steps outlined in the uninstall section of the edge-lb version documentation you are using`

Reply:
When we release version 1.3, etc, it will have its own set of release notes, just as 1.1 and 1.0 have their own release notes. We do not combine release notes from different releases into one document.